cmd/zpool/zpool_main.c

```diff
@@ -7728,6 +7955,25 @@ status_callback(zpool_handle_t *zhp, void *data)
 		    "complete.\n"));
 		break;
 
+	case ZPOOL_STATUS_REBUILDING:
+		printf_color(ANSI_BOLD, gettext("status: "));
+		printf_color(ANSI_YELLOW, gettext("One or more devices is "
+		    "currently being rebuilt.  The pool will\n\tcontinue "
+		    "to function, possibly in a degraded state.\n"));
+		printf_color(ANSI_BOLD, gettext("action: "));
+		printf_color(ANSI_YELLOW, gettext("Wait for the rebuild to "
+		    "complete.\n"));
+		break;
+
+	case ZPOOL_STATUS_REBUILD_SCRUB:
+		printf_color(ANSI_BOLD, gettext("status: "));
+		printf_color(ANSI_YELLOW, gettext("One or more devices have "
+		    "been rebuilt, scrubbing the pool is recommended.\n"));
+		printf_color(ANSI_BOLD, gettext("action: "));
+		printf_color(ANSI_YELLOW, gettext("Use 'zpool scrub' to "
```

Comment:
Maybe include the pool name in the `Use 'zpool scrub'` printf, since you could have multiple pools but only one needs to scrubbed.

Reply:
The status is printed per pool, so my feeling is there's no much to room for confusion. I tried to model the message after the other "action" messages for consistency.

Reply:
Ah, you're right - forgot about that.